tests/TestCase/Routing/RouteCollectionTest.php

```diff
@@ -37,7 +37,7 @@ public function setUp()
      * Test parse() throws an error on unknown routes.
      *
      * @expectedException \Cake\Routing\Exception\MissingRouteException
-     * @expectedExceptionMessage A route matching "/" could not be found
+     * @expectedExceptionMessage A "" route matching "/" could not be found
```

Comment:
I definitely agree... But can that happen when the framework is actually being used, rather than just tested? When I put that in the test, I thought that it would only ever appear in the test cases that call e.g. `$this->collection->parse('/');` directly, without including a method.

If there's a real situation where this could occur, then a different message would indeed be better. The simplest option might be to just remove the quotes so that it is either `A POST route matching...` or `A␣␣route matching...` (with a double space, that won't ever appear when rendered in HTML).